R/use_python.R

```diff
@@ -260,7 +260,9 @@ find_conda <- function() {
       path.expand("~/anaconda/bin/conda"),
       path.expand("~/anaconda2/bin/conda"),
       path.expand("~/anaconda3/bin/conda"),
-      path.expand("~/anaconda4/bin/conda")
```

Comment:
That's there for if/when there is an anaconda 4.

It seems like we'd also want the system level conda paths here? (i.e. just like https://github.com/rstudio/reticulate/blob/9e02a168788962539e83d1ea0fb85f084e85b274/R/config.R#L238)

Reply:
Oh yeah. I just added a commit that includes some missing ones.